model.py

```diff
@@ -9663,7 +9664,10 @@ def update(self, _db, equivalent_entries=None):
                 self.annotation = annotations[0]
 
         # Reset the entry's edition to be the Work's presentation edition.
-        best_edition = work.presentation_edition
+        if work:
+            best_edition = work.presentation_edition
+        else:
+            best_edition = None
```

Comment:
This happened in real usage with NYPL's staff picks list (we ended up with a Work that had no presentation edition, possibly due to an interaction with the best-seller list).